app/views/map/index.html.haml

```diff
@@ -49,8 +49,8 @@
           Kliknutím na zvýrazněné objekty se zobrazí
           %br
           relevantní dokumenty z úředních desek.
-      /.modal-footer
-      /  %button.btn.btn-primary{"data-dismiss" => "modal", :type => "button"} Rozumím
+      .modal-footer
+        %button.btn.btn-primary{ "data-dismiss" => "modal", :type => "button", :onclick => "disableWelcomeModalDialog()" } Nezobrazovat tento dialog

```

Comment:
IMHO by bylo lepší tam dát checkbox: `[ ] příště nezobrazovat` (klidně hodně menčím fontem).
a nechat tam button RozumÍm. Takhle to muze mast lidi (i) když to chtějí jen zrušít bez jakékoliv akce (ii) všichni by klikli na zruseni a uz by to nikdy nevideli.

Pritom z UX pruzkumu vyslo najevo, ze dobre, aby to tam mel kazdy alespon 2-3x.